app/fibo.hs

```diff
@@ -0,0 +1,15 @@
+fibo :: Integer -> Integer
```

Comment:
this solution is incomplete. please read the problem statement and the additional notes that have been added at http://www.vacationlabs.com/haskell/basic-types-and-functions.html#even-fibonacci-numbers You have to implement this using the two techniques mentioned therein.